packages/rancher-cis-benchmark/charts/values.yaml

```diff
@@ -5,7 +5,7 @@
 image:
   cisoperator:
     repository: rancher/cis-operator
-    tag: v1.0.10
+    tag: v1.0.10-rc1
```

Comment:
The tag should be 1.0.11-rc1
1.0.10 is already released

Reply:
bumped the cis-operator version